internal/mage/sdk/elixir.go

```diff
@@ -142,7 +143,45 @@ func (Elixir) Generate(ctx context.Context) error {
 
 // Publish publishes the Elixir SDK
 func (Elixir) Publish(ctx context.Context, tag string) error {
-	return nil
+	c, err := dagger.Connect(ctx, dagger.WithLogOutput(os.Stderr))
+	if err != nil {
+		return err
+	}
+	defer c.Close()
+
+	var (
+		version     = strings.TrimPrefix(tag, "sdk/elixir/v")
+		versionFile = "sdk/elixir/VERSION"
+		hexAPIKey   = os.Getenv("HEX_API_KEY")
```

Comment:
I have set the `HEX_API_KEY` secret in GitHub Actions. Can you please add https://hex.pm/users/dagger as one of the package owners?

Reply:
@gerhard I changed the package name into `dagger`. So I’m pretty sure that it would publish as a new package. :)

Ref the package name https://github.com/dagger/dagger/pull/5286/files#diff-2f6c105a25b9ac2d9564d7d923f2cd8034393081a21d41222c47bfa656f19c1fR9